recipes/expansionhunter/meta.yaml

```diff
@@ -1,47 +1,53 @@
 {% set name = "ExpansionHunter" %}
-{% set version = "4.0.2" %}
-{% set sha256 = "cbe66f09b8d86499966ae6da8655de04976e097227656495644fde4a69104525" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "921d1f8be658f2bdd144150278210bb921d605af921816e94a7508c0b9b9dfff" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
-
-build:
-  number: 1
-
 source:
   url: https://github.com/Illumina/ExpansionHunter/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - brackets.patch  # [osx]
-
+    - 0001-adjust-cmake-targets.patch
```

Comment:
Those are a lot of patches, can you please make sure that they find their way upstream :pray:  
Super nice work!

Reply:
I mentioned this a little in the PR description, but basically, the upstream package is (1) a tool by Illumina; (2) no longer under the Apache license on head; and (3) apparently orphaned in favor of a newer tool. I'm really hesitant to engage with it upstream. Pending more boost changes (see below), I don't think more patches will be required. I hope. Always hard to tell. If this is a blocker, I can withdraw the PR and just use my own channel's version of the package, don't want to make this unmaintainable.